compromises/2019/shadowhammer.md

```diff
@@ -0,0 +1,21 @@
+# ShadowHammer
+
+The update servers of ASUS company were compromised and
+a signed backdoored version of an application called 
+ASUS Live Update Utility was distributed to the ASUS users. 
+The application comes preinstalled on many Windows computers
+made by ASUS and is used to deliver updates for BIOS/UEFI firmware,
+hardware drivers and other ASUS tools.
+
+
+## Impact
+
+Over a million users might have downloaded and installed a 
+backdoored version of the application. For example, a report by
+Kaspersky shows over 57,000 Kaspersky users have installed
+the backdoored version of ASUS Live Update Utility. 
```

Comment:
It might be interesting to mention that, the attack seemed targeted towards (at least) 600 specific systems, whose mac addresses were hardcoded to receive a secondary payload.

Reply:
Good point. Added!